Modix.Data/Repositories/EmojiRepository.cs

```diff
@@ -277,7 +252,8 @@ limit 1
                     )
                     select ""EmojiId"", ""EmojiName"", ""IsAnimated"", ""Uses"", ""Rank"", ""TopUserId"", ""TopUserUses""
                     from stats, user_stats
-                    where { (emoji.Id is null ? @"""EmojiName"" = :EmojiName" : @"""EmojiId"" = :EmojiId") }";
+                    where {emojiClause}";
```

Comment:
Should this go up in the `stats` subquery, before the `GROUP BY` clause?

Reply:
I don't think so. I think if I do that, then the rank will always just be 1.

Reply:
Ah, yeah. The `Rank` column is what makes that necessary.